conf/routes

```diff
@@ -34,7 +34,7 @@ GET  /us/contribute                                 controllers.Application.cont
 GET  /au/contribute                                 controllers.Application.contributionsLanding(title="Support the Guardian | Make a Contribution", id="contributions-landing-page-au", js="contributionsLandingPageAU.js")
 GET  /eu/contribute                                 controllers.Application.contributionsLanding(title="Support the Guardian | Make a Contribution", id="contributions-landing-page-eu", js="contributionsLandingPageEU.js")
 
-GET  /contribute/recurring                          controllers.RegularContributions.displayForm(paypal: Option[Boolean])
+GET  /contribute/recurring                          controllers.RegularContributions.displayForm(paypal: Option[Boolean], directDebit: Option[Boolean])
```

Comment:
When did we delete this option?

Reply:
When I introduced the test because I didn't think it would be needed any more, but it makes it much easier to show the work to people so I put it back in.